Source/Scene/GlobeDepth.js

```diff
@@ -2,14 +2,10 @@ import BoundingRectangle from "../Core/BoundingRectangle.js";
 import Color from "../Core/Color.js";
```

Comment:
`SceneFramebuffer` and `PickDepth` can't be trimmed down much further but there's still a lot of room for improvement in `GlobeDepth`. I'm mainly thinking about the render state and scissor code which is in four different files and somewhat complex and mistake prone. I don't have a clear idea of how this might be integrated in `FramebufferManager`, if at all, but keep it in mind as you update the rest of the code.